inst/rmarkdown/templates/oup/skeleton/skeleton.Rmd

```diff
@@ -23,69 +23,75 @@ author:
         - 2
 address:
   - code: ABC
-    address: Address ABC
+    department: Department of Biology
+    organization: University of Idaho
+    street: 123 Main Street
+    #postcode: 
+    city: Boise
+    state: Idaho 07458 # Mix and match elements to get proper address
+    country: United States
   - code: DEF
-    address: Address DEF
+    address: Address in one element deprecated, but works
   - code: Another University
     address: Address for Another University
 footnote:
   - code: 1
     text: "Equal contribution"
   - code: 2
-    text: "Current email address: \\href{mailto:cat@example.com}{cat@example.com}"
-abstract: |
-  This is the abstract.
-
-  It consists of two paragraphs.
+    text: "Current email address: [cat@example.com](mailto:cat@example.com)"
+abstract:
+  - "This is the first paragraph of the abstract. It has several sentences making it go over several lines. For this it needs to have a lot of text." 
+  - "This is the second paragraph."
+  - "**Motivation:** You can also have some paragraphs start with bold face."
+keywords:
+  - keyword1
+  - keyword2
+  - keyword3
+#boxedlist: # Not fully compatible with all document formats
+#  - First point
+#  - Second point
+competing_interests: |
+  There are no competing interest.
+author_contributions: |
+  AA did all the work. The others are just freeloaders.
 acknowledgements: |
   This is an acknowledgement.
 
   It consists of two paragraphs.
-keywords:
-  - key
-  - dictionary
-  - word
-#fontsize: 12pt
-#spacing: halfline # could also be oneline
 #classoptions:
```

Comment:
Should the be example of classoptions to set ? 

The bundled template with OUP package seems to show several examples 

````latex
%%%CONTEMPORARY%%%
\documentclass[unnumsec,webpdf,contemporary,large]{oup-authoring-template}%
%\documentclass[unnumsec,webpdf,contemporary,large,namedate]{oup-authoring-template}% uncomment this line for author year citations and comment the above
%\documentclass[unnumsec,webpdf,contemporary,medium]{oup-authoring-template}
%\documentclass[unnumsec,webpdf,contemporary,small]{oup-authoring-template}

%%%MODERN%%%
%\documentclass[unnumsec,webpdf,modern,large]{oup-authoring-template}
%\documentclass[unnumsec,webpdf,modern,large,namedate]{oup-authoring-template}% uncomment this line for author year citations and comment the above
%\documentclass[unnumsec,webpdf,modern,medium]{oup-authoring-template}
%\documentclass[unnumsec,webpdf,modern,small]{oup-authoring-template}

%%%TRADITIONAL%%%
%\documentclass[unnumsec,webpdf,traditional,large]{oup-authoring-template}
%\documentclass[unnumsec,webpdf,traditional,large,namedate]{oup-authoring-template}% uncomment this line for author year citations and comment the above
%\documentclass[unnumsec,namedate,webpdf,traditional,medium]{oup-authoring-template}
%\documentclass[namedate,webpdf,traditional,small]{oup-authoring-template}
````

Reply:
These are already in the template and skeleton:

* `unnumsec` => `number_sections` YAML entry
* `contemporary`, `modern`, etc. => `document_style` YAML entry
* `large`, `medium`, etc. => `papersize` YAML entry

I haven't done anything with the `webpdf` option as that is always given in all the examples they show and in reality I am not 100% sure what it does.

Reply:
Oh ok. I missed that. I need to recheck how this works.